apps/ejabberd/include/mod_muc_light.hrl

```diff
@@ -75,13 +75,17 @@
           raw_config = [] :: raw_config()
          }).
 
+-type op_config() :: #config{}.
```

Comment:
When I first used it, it meant "operation" but now I'm not so convinced it's the best name. `muc_light_config()` is soooooo long. :D